src/Command/ErrorFormatter/TeamcityErrorFormatter.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PHPStan\Command\ErrorFormatter;
+
+use PHPStan\Command\AnalysisResult;
+use PHPStan\Command\Output;
+use PHPStan\File\RelativePathHelper;
+
+/**
+ * @see https://www.jetbrains.com/help/teamcity/build-script-interaction-with-teamcity.html#Reporting+Inspections
+ */
+class TeamcityErrorFormatter implements ErrorFormatter
+{
+
+	private RelativePathHelper $relativePathHelper;
+
+	public function __construct(RelativePathHelper $relativePathHelper)
+	{
+		$this->relativePathHelper = $relativePathHelper;
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function formatErrors(AnalysisResult $analysisResult, Output $output): int
+	{
+		$result = '';
+		$fileSpecificErrors = $analysisResult->getFileSpecificErrors();
+		$notFileSpecificErrors = $analysisResult->getNotFileSpecificErrors();
+		$warnings = $analysisResult->getWarnings();
+
+		if (empty($fileSpecificErrors) && empty($notFileSpecificErrors) && empty($warnings)) {
```

Comment:
`empty` isn't allowed as the build will tell you. Use more specific `count($fileSpecificErrors) === 0` instead.

Reply:
Yeah I saw it :-) It's fixed.